src/Microsoft.Tye.Core/ApplicationFactory.cs

```diff
@@ -117,21 +124,12 @@ public static async Task<ApplicationBuilder> CreateAsync(OutputContext output, F
                 builder.Services.Add(service);
 
                 // If there are no bindings and we're in ASP.NET Core project then add an HTTP and HTTPS binding
-                if (configService.Bindings.Count == 0 &&
-                    service is ProjectServiceBuilder project2 &&
-                    project2.IsAspNet)
+                if (configService.Bindings.Count == 0 && service is ProjectServiceBuilder project2 && project2.IsAspNet)
```

Comment:
Why did the formatting change here?

Reply:
Not sure, I don't remember touching such part. But also fixed back.